src/Game/UI/Gumps/OptionsGump.cs

```diff
@@ -2848,7 +2848,8 @@ private void Apply()
             ProfileManager.CurrentProfile.TooltipDisplayZoom = _tooltip_zoom.Value;
             ProfileManager.CurrentProfile.TooltipFont = _tooltip_font_selector.GetSelectedFont();
 
-            ProfileManager.CurrentProfile?.Save(UIManager.Gumps.OfType<Gump>().Where(s => s.CanBeSaved).Reverse().ToList());
+            ProfileManager.CurrentProfile?.Save
+                (ProfileManager.ProfilePath, UIManager.Gumps.OfType<Gump>().Where(s => s.CanBeSaved).Reverse().ToList());
```

Comment:
Maybe you could store the `ProfileManager.CurrentProfile` calls into a `var currentProfile` variable.

Reply:
Sorry, I don't have any experience in C#.  Can you point me to documentation on this feature?  (I'm an embedded linux driver developer in C).

Reply:
In this file you have quite a lot of `ProfileManager.CurrentProfile.` occurences which make certain lines rather long.
Instead of calling these all the time, create a local variable `Profile currentProfile = ProfileManager.CurrentProfile;` and use that one instead.

Reply:
Ahh, ok.  I completely misunderstood that.

Reply:
Fixed.